### ISSUE TITLE
Not working with Electrolysis

### ISSUE THREAD

**Issue by NicolasWebDev** [jkoomjian/PageZipper#1]
Hi,

I am using FF developer edition (version 49.0a2 as of today) on ArchLinux.
The FF extension is not working when I enable the "Multi process firefox developer edition", aka Electrolysis, aka e10s.
There is no way to load more pages when clicking on the right arrow.
Going to the page end does not load the next one either.

Thanks for the great plugin though.

Cheers

**Comment by jkoomjian** [comment]
Yeah, Firefox 49 no longer supports the the Overlay api the PageZipper extension uses. I'll update the extension to use the newer api. Thanks for letting me know.

**Comment by NicolasWebDev** [comment]
Thanks @jkoomjian for the answer I was not really hoping for, because it is the unique issue on this tracker ^^.

The Overlay API is disabled when using Electrolysis? Because when disabling Electrolysis, it works for me with FF 49.

**Comment by jkoomjian** [comment]
If I understand the documentation correctly, the Electrolysis/multiprocess architecture is the future of Firefox, and the Overlay Api won't be supported on it: https://developer.mozilla.org/en-US/Add-ons/Working_with_multiprocess_Firefox. So the extension needs to be updated. Fortunately the new webextension api is pretty much the same as the chrome extension api, and PageZipper already works on chrome :)
